docs/changelog.md

```diff
@@ -11,6 +11,10 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* ****concept-engine****, **browser-session**
+  * ⚠️ **This is a breaking change**: Function signature changed  `Session.Observer.onTrackerBlocked(session: Session, blocked: String, all: List<String>) = Unit` from to `Session.Observer.onTrackerBlocked(session: Session, tracker: Tracker, all: List<Tracker>) = Unit`
```

Comment:
Let's maybe also add what we added e.g. that we now provide more details about a blocked content tracker, not just the changed method signature.

Reply:
👍

Reply:
Added more info :)